docker-build.sh

```diff
@@ -5,6 +5,123 @@
 set -o errexit
 set -o xtrace
 
+ARCHIVE_ADDR=https://archive.ubuntu.com/ubuntu/
+PORTS_ADDR=https://ports.ubuntu.com/
+
+# Prepare HWA headers, libs and drivers for x86_64-linux-gnu
+prepare_hwa_amd64() {
+    # Download and install the nvidia headers from deb-multimedia
+    pushd ${SOURCE_DIR}
+    git clone --depth=1 https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+    pushd nv-codec-headers
+    make
+    make install
+    popd
+    popd
+
+    # Download and setup AMD AMF headers from AMD official github repo
+    # https://www.ffmpeg.org/general.html#AMD-AMF_002fVCE
+    svn checkout https://github.com/GPUOpen-LibrariesAndSDKs/AMF/trunk/amf/public/include
+    pushd include
+    mkdir -p /usr/include/AMF && mv * /usr/include/AMF
```

Comment:
```suggestion
    mkdir -p /usr/include/AMF
    mv * /usr/include/AMF
```
this reads cleaner IMHO and separate `&&` is not needed because the script has `errexit` set and `mkdir -p` doesn't return non-zero if folder exists

Reply:
> I am not quite sure why we need to maintain another ffmpeg docker image.

Because legacy reasons. I think we should use single ffmpeg, too.